docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -51,11 +51,71 @@ This also prevents users signing over any hashed transaction data (fee, transact
 
 We propose to maintain functional tests using bijectivity in the SDK.
 
-### 2. Only ASCII 32-127 characters allowed
+### 2. UTF-8 characters allowed, but signing devices MAY convert them before display
 
-Ledger devices have limited character display capabilities, so all strings MUST only contain ASCII characters in the 32-127 range.
+The SIGN_MODE_TEXTUAL specification allows all UTF-8 characters. The textual strings will contain all characters as-is, with the modifications below:
+- the line feed `\n` character (ASCII: 10) is escaped using quotation marks: `"\n"`. This is to disambiguate with the `\n` control character used to signal a screen change on the signing device.
+- the quotation mark character `"` (ASCII: 34) is escaped with a backslash prefix: `\"`. This is to allow bijectivity if the signing device decides to convert UTF-8 characters into its own set of displayable characters, using `"` as a control character.
```

Comment:
It's actually the backslash character `\`, not quotation marks, that needs to be quoted in order to legibly and bijectively quote newlines.  As written, this allows all other ASCII control characters, e.g. nul, bel, etc, to transmit as themselves despite having no printable representation, even on Unicode-enabled devices.

I strongly recommend following an established standard quotation algorithm rather than trying to invent something new. There are a large number of existing designs that follow the basic pattern of backslash followed by:

- `b`: backspace
- `f`: form feed
- `n`: newline
- `r`: carriage return
- `t`: horizontal tab
- `v`: vertical tab
- `0`: nul
- `\`: backslash
- some means of quoting other control characters by number

These transformations ought to be part of the textual spec, rather than relegated to the Ledger, because they're necessary for bijectivity and accurate legible quoting.

Reply:
Oh ok. I completely misunderstood you during our monday call, when you were mentioning "quotation" i understood quotation marks, hence this PR. A lot of this needs to be rewritten then.